forwarder/kernel-forwarder/pkg/kernelforwarder/remote/vxlan.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Please keep the license of the code if you move it around.

Reply:
Done